internal/namespaces/instance/v1/ip_test.go

```diff
@@ -0,0 +1,59 @@
+package instance
+
+import (
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+)
+
+func Test_IpCreate(t *testing.T) {
+	t.Run("Simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance ip create",
+		Check:    core.TestCheckGolden(),
```

Comment:
most `Check` should have a `TestCheckExitCode`

we can use `TestCheckCombine` to have multiple checks